.github/workflows/code-engine-preview.yml

```diff
@@ -62,7 +76,7 @@ jobs:
         with:
           step: start
           token: ${{ github.token }}
-          env: preview-${{ env.PR_TAG }}
+          env: ${{ inputs.env_name }}
```

Comment:
Correct me if I'm wrong @abdonrd @vabarbosa. The idea now is to have:

- For PRs we are going to configure:
  - `preview_tag` won't be set
  - `env_name`: it seems we are not going to need to set it but then it disappears the `preview-PR`. This format is correct?
- For `main` branch we are going to configure the next variables:
  - `preview_tag`: staging
  - `env_name`: staging

Reply:
correct, that is the idea... all PR previews will be in a `Previews` environment
and for `main` branch it would be in `Staging` environment and `staging` tag so the URL would be `saiba-staging.uuid.us-south.codeengine.appdomain.cloud`